darwin/importer/importer.py

```diff
@@ -697,22 +723,22 @@ def _handle_annotators(
 def _get_annotation_data(
     annotation: dt.AnnotationLike, annotation_class_id: str, attributes: dt.DictFreeForm
 ) -> dt.DictFreeForm:
+    # Todo Convert to import Darwin format
```

Comment:
Is this still a TODO?

Reply:
Not anymore, since I did that now (hopefully)